translations/fr.json

```diff
@@ -0,0 +1,224 @@
+        "issue": "Problèmes",
+        "issue_desc": "Envoyez vos problèmes aux développeurs",
+        "issue_message_desc": "Écrivez votre message",
+        "level": "niveau",
+        "link": "lien",
+        "map": "carte",
+        "map_desc": "Obtenez des informations détaillées sur une carte",
+        "map_select_desc": "Sélectionnez une carte",
+        "message": "message",
+        "name": "nom",
+        "name_search_desc": "Nom de l'objet à rechercher",
+        "patchnotes": "notes de patch",
+        "patchnotes_desc": "Obtenez les dernières notes de patch",
+        "price": "prix",
+        "price_desc": "IDk",
+        "progress": "IDK",
+        "progress_desc": "IDK",
+        "progress_flea_market_fee_desc": "Définissez votre progression pour calculer avec précision les frais de marché aux puces",
+        "progress_flea_market_fee_intel_select_desc": "Niveau du centre de renseignement",
+        "progress_flea_market_fee_mgmt_select_desc": "Niveau de compétence en gestion du cachette",
+        "progress_hideout_desc": "IDK",
+        "progress_hideout_level_select_desc": "IDK",
+        "progress_hideout_station_select_desc": "IDK",
+        "progress_level_desc": "IDK",
+        "progress_level_select_desc": "IDK",
+        "progress_link_desc": "Liez votre compte TarkovTracker pour synchroniser la progression de votre refuge.",
+        "progress_link_token_desc": "Votre jeton API TarkovTracker sur https://tarkovtracker.io/settings/",
+        "progress_show_desc": "Affiche votre progression personnalisée",
+        "progress_skill_desc": "Définir le niveau de compétence",
+        "progress_skill_level_select_desc": "Le niveau de la compétence",
+        "progress_trader_desc": "Définir le niveau du marchand",
+        "progress_trader_level_select_desc": "Le niveau du marchand",
+        "progress_unlink_desc": "Délier votre compte TarkovTracker",
+        "restock": "réapprovisionnement",
+        "restock_alert_desc": "Définir des alertes pour les réapprovisionnements des commerçants",
+        "restock_alert_send_desc": "Envoi ou non d'une alerte",
+        "restock_channel_select_desc": "Le canal sur ce serveur dans lequel le Bot fera des annonces",
+        "restock_channel_desc": "Annoncer les réapprovisionnements des marchands dans un canal Discord",
+        "restock_desc": "Afficher ou définir des alertes pour les délais de réapprovisionnement des marchands",
+        "restock_show_desc": "Afficher les délais de réapprovisionnement des marchands",
+        "roulette": "roulette",
+        "roulette_desc": "Faites tourner la roulette pour un jeu amusant de Tarkov !",
+        "send_alert": "IDk",
+        "show": "montrer",
+        "skill": "compétence",
+        "skill_desc": "Compétence",
+        "station": "station",
+        "status": "statut",
+        "status_desc": "Vous donne l'état actuel du serveur",
+        "stim": "IDK",
```

Comment:
"stim" in all contexts here is shorthand for "stimulant". In EFT, a stimulant is an in-game item that players use by injecting themselves with chemicals which give various in-game enhancements. Such as carrying more loot, running faster, or stopping bleeding